### ISSUE TITLE
MD060 table format does not trigger

### ISSUE THREAD

**Issue by o-l-a-v** [rvben/obsidian-rumdl#12]
## Env

* Windows 11 x64
* Obsidian v1.11.5
* rumdl Obsidian extension installed with BRAT v0.0.30

## Problem

Ref.:

* <https://github.com/rvben/rumdl/issues/344>
* <https://github.com/rvben/rumdl/blob/main/docs/md060.md>

I now have a config file in the root of my vault that the rumdl Obsidian extension detects.

```toml
# rumdl configuration
# Generated from Obsidian plugin settings
# Schema: https://raw.githubusercontent.com/rvben/rumdl/main/rumdl.schema.json

[global]
disable = ["MD013", "MD025", "MD041"]

[MD003]
style = "atx"

[MD004]
style = "asterisk"

[MD060]
enabled = true
style = "aligned"
```

I have a note in raw markdown edit mode, and do CTRL + S to format on save.

```md
|col1|col2|col3|
|:--|:-:|--:|
|hi|ho|letsgo|
|hihi|hoho|letsgogo|
```

Nothing happens.
